install_config/storage_examples/privileged_pod_storage.adoc

```diff
@@ -0,0 +1,205 @@
+= Mount Volumes on Privileged Pods
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap:
+
+== Overview
+
+The following provides a method for mounting persistent volumes to pods attached
+to the *privileged scc*.  For the purpose of this guide, GlusterFS will be the
+sample use-case though this method can be adapted to any
+link:../install_config/persistent_storage/index.html[supported storage configuration].
+For further simplicity, we will be using the
+link:https://docs.openshift.com/enterprise/3.1/install_config/configuring_authentication.html#AllowAllPasswordIdentityProvider[AllowAllPasswordIdentityProvider]
+authentication method.
```

Comment:
Is this step required? I set it for east on the UI, but I would suggest you try it without setting this wide open and see if it still works.